app/client/cli/debug.go

```diff
@@ -210,51 +190,16 @@ func sendDebugMessage(cmd *cobra.Command, debugMsg *messaging.DebugMessage) {
 	}
 
 	// if the message needs to be broadcast, it'll be handled by the business logic of the message handler
+	//
+	// DISCUSS_THIS_COMMIT: The statement above is false. Using `#Send()` will only
+	// be unicast with no opportunity for further propagation.
 	validatorAddress = pstore.GetPeerList()[0].GetAddress()
```

Comment:
Unrelated. Thoughts on renaming `validatorAddress` to `randomStakedActorAddress` or `firstStakedActorAddress`?